test/run_cargo_test.ps1

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env powershell
+
+#Requires -Version 5
```

Comment:
Let me see if I can get that to work - that seems like a good idea.

Reply:
After thinking about this a bit... There's some upsides and downsides.
My design for this was essentially to try and have parity between the linux and windows scripts. In this design, the script assumes the *only* thing you need is bash/powershell and habitat and it will install all the build deps.

The other option assumes that we have hab and powershell, and would also add an extra "passthrough" step for the parameterized bat file (or pushing the whole invocation up to pipeline yml).

Personally I like the existing approach better but let me know if there's anything else I'm missing. I think documenting the requirements should be sufficient.

Reply:
That's cool. This only becomes an issue on a windows system < 2016 (unless you explicitly update the WMF). So if we start getting complaints about that (which I very much doubt we will) then this is fine.